us/en/skymiles/blocks/accordion/accordion.js

```diff
@@ -0,0 +1,100 @@
+export default async function decorate(block) {
+  const accordions = [...block.children];
+  accordions.forEach((accordion) => {
+    accordion.classList.add('accordion-section');
+  });
+
+  const parents = block.querySelectorAll('.accordion-section');
+  parents.forEach((({ children }) => {
+    children[0].classList.add('header');
+    children[0].setAttribute('aria-expanded', 'false');
+    children[1].classList.add('text');
+  }));
+
+  // creating buttons in text divs
+  const textDiv = block.querySelectorAll('.text');
+  textDiv.forEach((text) => {
+    const textButton = text.querySelectorAll('p > small > strong');
+    textButton.forEach((tb) => {
+      const button = document.createElement('button');
+      button.textContent = tb.textContent;
+      const p = tb.closest('p');
+      p.parentNode.insertBefore(button, p);
+      p.parentNode.removeChild(p);
+    });
+  });
+
+  const parentWrapper = document.querySelectorAll('.accordion-wrapper');
+  parentWrapper.forEach((wrapper) => {
+    if (!wrapper.querySelector('.toolbar')) {
```

Comment:
You can change this to a guard clause to save on the indentation:
```
if (wrapper.querySelector('.toolbar')) {
  return;
}

…
```

Reply:
unable to follow this particular one

Reply:
The idea of a _guard clause_ is just to reverse the conditional and exit early. that way you avoid indenting the main code and it usually makes it easier to read and maintain as the core of the method is essentially fully contained at the end in its own block.

It's nothing blocking the PR, but just a best practice that usually helps with long-term maintainability.

If you are not familiar with that pattern, here are a few quick examples:
- https://www.codementor.io/@clintwinter/use-guard-clauses-for-cleaner-code-1rrsczgwxp
- https://www.carloscaballero.io/refactoring-guard-clauses/